mautic/mautic
# Default visibilty of filter area in reports

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| New feature? | X

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
While taking report default it is set to 1 month, but since the filter area is collapsed user may feel confused. So filter needs to be visible(expanded) as default.
![report-collapsed](https://cloud.githubusercontent.com/assets/24408961/22974114/e4b7a0ae-f3a6-11e6-919d-a30cf190dba9.png)
![report](https://cloud.githubusercontent.com/assets/24408961/22974115/e4bebcc2-f3a6-11e6-9790-2eefafe52441.png)


#### Steps to test this PR:
1. Create report in Report page
2. View the report 
3. check whether the filter area is visible on page load
